universe/world/users.go

```diff
@@ -152,7 +152,7 @@ func (w *World) noLockRemoveUser(user universe.User, updateDB bool) (bool, error
 	for _, child := range w.allObjects.Data {
 		if child.LockUIObject(user, 0) {
 			w.Send(
-				posbus.WSMessage(&posbus.LockObjectResponse{ID: child.GetID(), State: 0, LockOwner: user.GetID()}),
+				posbus.WSMessage(&posbus.LockObjectResponse{ID: child.GetID(), Result: 0, LockOwner: user.GetID()}),
```

Comment:
hm, should the owner here be nil as well?

Reply:
yes, refresh. I just beat you by 15seconds :p new commit on the PR

Reply:
no you didn't - it's a different place :p

Reply:
d'oh, yeah, cleanup function